Use implicit animations for animatable properties.

squall77it opened the pull request:
Tweener was replaced with Clutter implicit animations.

squall77it commented:
Sorry I want to make a second pull request with Italian translation, but the commit was automatically added to this pull request, i revert it. When this is closed i make a new one. I'm not very good with git.

neumann-d commented on ShutdownTimer@neumann/extension.js:
```diff
@@ -49,11 +50,11 @@ function _showTextbox(textmsg) {
+		opacity: 0,
+		duration: 4000,
+		mode: Clutter.AnimationMode.EASE_OUT_QUAD,
+		onComplete:  () => { _hideTextbox; }
```
```suggestion
	 textbox.ease ({
		opacity: 0,
		duration: 4000,
		mode: Clutter.AnimationMode.EASE_OUT_QUAD,
		onComplete:  _hideTextbox
	});
```

neumann-d commented:
Thank you very much @squall77it  for this pull request. I'm still running GNOME 3.36, but your code change also works here. I just got some Syntax Error because of missing closing brackets. See my suggestion. When you update your Pull Request, I will merge it :+1:

squall77it commented:
Thank you for the great extension. Sorry for the  syntax error, but when i cleaned  the file I deleted one line too much.
I have updated the pull request.

neumann-d commented:
Thank you  @squall77it for contributing to this extension
